.chloggen/elasticsearchexporter-dynamic-index.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
```

Comment:
I think this could be new `feature` 👍

Reply:
@JaredTan95 I conducted the points you mentioned. check it out :)

Reply:
@JaredTan95 
```
@github-actions
build-and-test / build-package (deb) (pull_request) Failing after 2m
[Details](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3656753677/jobs/6180031352)
@github-actions
build-and-test / build-package (rpm) (pull_request) Failing after 3m
```
looks like the pipeline or the build process is wrong.